src/PolicyTool/Program.cs

```diff
@@ -0,0 +1,307 @@
+﻿//#define Attach
+using System;
+using Microsoft.Extensions.Configuration;
+using System.Net;
+using ReactiveDomain.Foundation;
+using ES = EventStore.ClientAPI;
+using ReactiveDomain.EventStore;
+using System.CommandLine;
+using ReactiveDomain.Messaging.Bus;
+using ReactiveDomain.Policy.Messages;
+using ReactiveDomain.Messaging.Messages;
+using System.Diagnostics;
+using System.Threading;
+using System.Linq;
+using ReactiveDomain.Policy;
+using ReactiveDomain.Users.Services;
+using System.DirectoryServices.AccountManagement;
+using ReactiveDomain.Users.ReadModels;
+using ReactiveDomain.Users.Messages;
+using ReactiveDomain.Policy.Domain;
+
+namespace PolicyTool
+{
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+    public static class Program
+    {
+        public static IConfiguredConnection EsConnection;
+        public static IConfigurationRoot AppConfig;
+
+
+        /// <summary>
+        /// Policy Setup Tool
+        /// </summary>
+
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Interoperability", "CA1416:Validate platform compatibility", Justification = "Program only valid on Windows with AD")]
+        static int Main(string[] args)
+        //public static int Main(string[] args)
```

Comment:
??

Reply:
the AD calls raise a warning about not working on linux

Reply:
What's up with the commented code?

Reply:
different variations on how `System.CommandLine` gets invoked